phpunit.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/7.5/phpunit.xsd"
+		backupGlobals="false"
+		backupStaticAttributes="false"
+		bootstrap="tests/bootstrap.php"
+		colors="true"
+		convertErrorsToExceptions="true"
+		convertNoticesToExceptions="true"
+		convertWarningsToExceptions="true"
+		processIsolation="false"
+		stopOnError="false"
+		stopOnFailure="false"
+		stopOnIncomplete="false"
+		stopOnSkipped="false"
+		verbose="true"
```

Comment:
Was it intentional for these lines to be more indented than 
`xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"` and
`xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/7.5/phpunit.xsd"` ?

Reply:
Not intentional. Copy-pasted from `wpseo-news` 🙂 Please do feel free to fix the indentation.